reactioncommerce/reaction
# Adjust inventory on shipment

Closes #964 
Implements two new status changes:

1. When an order is placed, moves inventory from "reserved" to "sold".
2. When an order is shipped moves inventory from "sold" to "shipped".

How to test:

1. Add items to cart
2. Check that an appropriate number of items are marked `workflow.status` = "reserved".
3. Finish your order
4. Verify those items are now marked `workflow.status` = "sold".
5. Log into the dashboard.
6. Ship the same order
7. Verify that the same items are now marked `workflow.status` = "shipped".

I think that `inventory/setStatus` probably needs to be restructured to handle different types of logic for different transition states (and to be able to extend it without modifying the existing code).
